ethereum/contracts/relayer/deliveryProvider/DeliveryProviderState.sol

```diff
@@ -54,3 +54,20 @@ contract DeliveryProviderStorage {
 contract DeliveryProviderState {
     DeliveryProviderStorage.State _state;
 }
+
+struct SupportedMessageKeyTypes {
+    uint256 bitmap;
+}
+
+//keccak256("DefaultDeliveryProviderState") - 1
```

Comment:
Can we be more specific here for the storage slot lookup? I imagine this contract's storage will grow in the future.

Reply:
Done